static/node/chain/chain.ethereum.js

```diff
@@ -108,9 +108,16 @@ async function startServer(options) {
   try {
     await server.listen(options.port, options.hostname);
   } catch (err) {
+    const { message, stack} = err;
+    let {code} = err;
+    // todo: we may be able to remove this check once https://github.com/trufflesuite/ganache/issues/4020 is resolved
+    if (code === undefined && message && message.indexOf("listen EADDRINUSE") === 0) {
```

Comment:
Should it be > 0? If it's supposed to be 0, then perhaps use `string.startsWith`?

Reply:
It should be `string.startsWith` because the error message explicitly is `listen EADDRINUSE: <reason>` - I'll change to that.

Thanks!